asyncdynamo/asyncdynamo.py

```diff
@@ -180,72 +178,81 @@ def raise_error():
         accept an error argument, which will be a boto.exception.DynamoDBResponseError.
 
         If there is not a valid session token, this method will ensure that a new one is fetched
         and cache the request when it is retrieved.
-        '''
+        """
         this_request = functools.partial(self.make_request, action=action,
             body=body, callback=callback,object_hook=object_hook)
         if self.authenticate_requests and self.provider.security_token in [None, PENDING_SESSION_TOKEN_UPDATE]:
-            # we will not be able to complete this request because we do not have a valid session token.
-            # queue it and try to get a new one. _update_session_token will ensure that only one request
+            # we will not be able to complete this request because we do not
+            # have a valid session token. queue it and try to get a new one.
+            # _update_session_token will ensure that only one request
             # for a session token goes out at a time
             self.pending_requests.appendleft(this_request)
+
             def cb_for_update(error=None):
                 # create a callback to handle errors getting session token
-                # callback here is assumed to take a json response, and an instance of DynamoDBResponseError
+                # callback here is assumed to take a json response,
+                # and an instance of DynamoDBResponseError
                 if error:
                     return callback({}, error=DynamoDBResponseError(error.status, error.reason))
                 else:
                     return
             self._update_session_token(cb_for_update)
             return
-        headers = {'X-Amz-Target' : '%s_%s.%s' % (self.ServiceName,
-                                                  self.Version, action),
-                'Content-Type' : 'application/x-amz-json-1.0',
-                'Content-Length' : str(len(body))}
-        request = HTTPRequest(self.url,
-            method='POST',
-            headers=headers,
-            body=body,
-            validate_cert=self.validate_cert)
-        request.path = '/' # Important! set the path variable for signing by boto (<2.7). '/' is the path for all dynamodb requests
-        request.auth_path = '/' # Important! set the auth_path variable for signing by boto(>2.7). '/' is the path for all dynamodb requests
+        body = body.encode('utf-8')
+        headers = {'X-Amz-Target': '%s_%s.%s' % (self.ServiceName,
+                                                 self.Version, action),
+                   'Content-Type': 'application/x-amz-json-1.0',
+                   'Content-Length': str(len(body))}
+        request = HTTPRequest(self.url, method='POST', headers=headers,
+                              body=body, validate_cert=self.validate_cert)
+        request.path = '/'  # Important! set the path variable for signing by boto (<2.7). '/' is the path for all dynamodb requests
```

Comment:
Why didn't you made this and next comment on a new line?

Reply:
The aim was to minimize notifications in PyCharm rather than make the code actually better.